packages/components/src/button/style.scss

```diff
@@ -263,15 +263,24 @@
 	}
 
 	&.is-small {
-		height: $icon-size;
+		height: $button-size-small-next-default-32px;
 		line-height: 22px;
 		padding: 0 8px;
 		font-size: 11px;
 
 		&.has-icon:not(.has-text) {
 			padding: 0;
-			width: $icon-size;
-			min-width: $icon-size;
```

Comment:
As @ciampo noted, these `$icon-size` variables should've been `$button-size-small` (both 24px though).

This error has also been fixed in this PR.